src/openrct2/localisation/StringIds.h

```diff
@@ -4004,6 +4004,9 @@ enum : uint16_t
 
     STR_SHORTCUT_TOGGLE_WALL_SLOPE = 6594,
 
+    STR_AUTHOR = 6595,
+    STR_AUTHOR_PLURAL = 6596,
```

Comment:
I think it's better to identify these as strings used on scenery, because authorship will also be shown elsewhere.
```suggestion
    STR_SCENERY_AUTHOR = 6595,
    STR_SCENERY_AUTHOR_PLURAL = 6596,
```

Reply:
Curious, couldn't these strings be reused for #20952?
If that's the case then the original strings would make more sense.

Reply:
As counter intuitive as it sounds, we typically don't reuse strings into other windows to avoid changing one later and not seeing that it affects the other